lib/core/common/helpers/validation_helper.dart

```diff
@@ -0,0 +1,17 @@
+class ValidationHelper {
+  static final RegExp _emailRegExp = RegExp(
```

Comment:
_emailRegExp -> emailRegExp. if using static variable do not named that as a private.